### PR TITLE
loss functions draft 

### PR DESCRIPTION
**Pull request by aarongrech79**

### CODE REVIEW

File: terms/lossfunctions_ag.bib [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  pages = 	 {37--45},
+  year = 	 {2014},
+  editor = 	 {Eric P. Xing and Tony Jebara},
+  volume = 	 {32},
+  number =       {1},
+  series = 	 {Proceedings of Machine Learning Research},
+  address = 	 {Bejing, China},
+  month = 	 {22--24 Jun},
+  publisher = 	 {PMLR},
+  pdf = 	 {http://proceedings.mlr.press/v32/yanga14.pdf},
```
Comment (jp-um) [inline terms/lossfunctions_ag.bib]: There are many useless entries here ... please remove.  See rule 8. in the guidelines.

File: terms/lossfunctions_ag.tex [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+\begin{marginfigure}%
+	\includegraphics[width=\linewidth]{loss_functions/cross_entropy.png}
+	\caption{Log Loss Function}
+	\label{fig:logloss}
+\end{marginfigure}
+
+
+The three canonical loss functions as described in Vapnik’s \citep{vapnik1998statistical} work are as follows:
```
Comment (jp-um) [inline terms/lossfunctions_ag.tex]: See point 26 in guidelines.  Did you see the pdf you generate?  Strange characters due to the single quote.

**Comment by jp-um** [comment]
Why did you close this?

**Comment by aarongrech79** [comment]
I had to close since I could not update it and commit any changes. However
I created another pull request.

**Comment by jp-um** [comment]
Ok merged.

*Dr Jean-Paul Ebejer**,*  BSc (Hons) (Melita), MSc (Imperial), DPhil (Oxon.)

*Lecturer*
Centre for Molecular Medicine and BioBanking,
Office 320, Biomedical Sciences Building,
University of Malta, Msida MSD 2080.
T: (00356) 2340 3263

<https://twitter.com/malteseunderdog>

**Pull request closed without merging**